src/week-view.c

```diff
@@ -36,8 +36,12 @@ struct _WeekView {
 	GSList* calendars;
 	EventWidget* events_week[7];
 	EventWidget* events_allday[7];
+	int todays_week;
+	int todays_year;
+	// TODO TBD: rename to "shown_week" and "shown_year" to prevent confusing the displayed with current week and year?
 	int current_week; // 1-based, note libical is 0-based
 	int current_year;
+	// TODO TBD: rename to "weekday_start" and "weekday_end" to prevent confusing int with UNIX timestamp?
```

Comment:
Not relevant to this change, but in general agreed. I'd perhaps suggest `span_dow_start`, if `dow` is clear enough for "day of week", otherwise `span_weekday_start` is also ok

Reply:
i wouldn't have understood dow, so i'd prefer span_weekday_start (but i am not a native english speaker)